pounders/py/general_h_funs.py

```diff
@@ -45,10 +45,12 @@ def emittance_h(F):
 
 
 def squared_diff_from_mean(Cres, Gres, Hres):
-    # Combines models for the following h function
-    #    h = @(F)sum((F - 1/m*sum(F)).^2) - alpha*(1/m*sum(F))^2
-    # That is, the objective is to have the vector close to it's mean, and have
-    # a small mean (penalized by alpha)
+    """
+    Combines models for the following h function
+       h = @(F)sum((F - 1/m*sum(F)).^2) - alpha*(1/m*sum(F))^2
```

Comment:
Is this MATLAB notation in a python docstring?

Reply:
It is, but it should be legible to most users interested in such functionality.